Classes/Aspects/AugmentationAspect.php

```diff
@@ -166,6 +170,6 @@ protected function needsMetadata(NodeInterface $node, $renderCurrentDocumentMeta
         /** @var $contentContext ContentContext */
         $contentContext = $node->getContext();
 
-        return ($contentContext->isInBackend() === true && ($renderCurrentDocumentMetadata === true || $this->nodeAuthorizationService->isGrantedToEditNode($node) === true));
+        return $contentContext->isInBackend() === true || $renderCurrentDocumentMetadata === true;
```

Comment:
note that we removed this $renderCurrentDocumentMetadata thingy in Neos 9.0 already, it is very much a code smell but i left it untouched :)